### PR TITLE
U/fjammes/dm 4642

### PR DESCRIPTION
**Pull request by fjammes**
My tests show that it is still compliant with MySQL.

### CODE REVIEW

File: bin/scisql-deploy.py [hunk 1]
```diff
@@ -13,16 +11,15 @@
...
 
 def parse_args():
 
     parser = argparse.ArgumentParser(
             description='''sciSQL deployment tool. Install sciSQL plugin in a
-MySQL running instance :\n
-- install shared library in MySQL plugin directory\n
-- install UDF in MySQL database''',
-            )
+Mariadb/MySQL running instance :\n
```
Comment (smonkewitz) [inline bin/scisql-deploy.py]: MariaDB not Mariadb (for consistency)

File: templates/test.sh [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
 do
-    echo -n "Running ${testfile} : "
-    if python ${testfile}; then
-        echo "PASS"
+    test_msg="Test ${testfile} :"
+    if python "${testfile}"; then
+        echo "${test_msg} SUCCESS"
     else
-        echo "FAIL"
+        echo "${test_msg} FAILURE"
```
Comment (smonkewitz) [inline templates/test.sh]: The way this was written, the name of the test being run is printed before the test is run, which is useful if the test hangs. Unless you have a good reason, I'd prefer it if you changed this back.

File: tools/mysqlversion.py [hunk 3]
```diff
@@ -2,56 +2,89 @@
...
-def __parse_version(version):
-    return tuple(map(int, version.split('.')))
+_CMP = {
+    _MIN_VERSION: operator.ge,
+    _EXACT_VERSION: operator.eq,
+    _MAX_VERSION: operator.le,
+}
+
+def _to_tuple(version):
+    print version.split('.')
```
Comment (smonkewitz) [inline tools/mysqlversion.py]: This print statement should be removed.

**Comment by fjammes** [comment]
Thanks Serge, I've taken in account all your remarks :+1:

**Comment by fjammes** [comment]
@smonkewitz, could you please merge this PR or give me write acces to this repos?

Thanks

File: templates/test.sh [hunk 4]
```diff
@@ -1,20 +1,21 @@
...
 
 retcode=0
 for testfile in ${SCISQL_BASE}/test/test*.py ${SCISQL_BASE}/tools/docs.py
 do
-    echo -n "Running ${testfile} : "
-    if python ${testfile}; then
-        echo "PASS"
+    test_msg="Test ${testfile} :"
+    echo "Running ${test_msg}"
+	if python "${testfile}"; then
```
Comment (smonkewitz) [inline templates/test.sh]: Indentation.

**Comment by smonkewitz** [comment]
This package is external to lsst, so I don't think it makes sense to have an ups repository in there. Can you remove it? It looks like maybe you intended to, given your commit message. Then I'll merge this.

**Comment by fjammes** [comment]
ok everything done, thanks for your help :-)

**Pull request merged**